src/components/dialog/dialog.style.js

```diff
@@ -0,0 +1,84 @@
+import styled, { css } from 'styled-components';
+import baseTheme from '../../style/themes/base';
+
+const dialogSizes = {
+  extraSmall: '300px',
+  small: '380px',
+  mediumSmall: '540px',
+  medium: '750px',
+  mediumLarge: '850px',
+  large: '960px',
+  extraLarge: '1080px'
+};
+
+const DialogStyle = styled.div`
+  background-color: #F2F5F6;
```

Comment:
should this value be hardcoded?

Reply:
I guess so, at least till we decide how to keep our colors

Reply:
but it's not the Classic style, right?

Reply:
If im not wrong it's like `Dialog` doesn't have different versions based on themes